plugins/frontend/backstage-plugin-jira/src/api/index.ts

```diff
@@ -217,4 +229,33 @@ export class JiraAPI {
       ),
     ];
   }
+
+  async getIssuesCountByJql(query: CustomQuery) {
+    const { apiUrl } = await this.getUrls();
+
+    // TODO : mutualize this ?
+    const request = await fetch(`${apiUrl}search`, {
```

Comment:
this is also used in the `getIssuesCountByType ` function, maybe we should create a dedicated function ?